crypto/BRCryptoBase.h

```diff
@@ -97,8 +97,12 @@ extern "C" {
 #define CRYPTO_REF_ASSIGN(release)  (BRCryptoRef) { 1, CRYPTO_AS_FREE (release) }
 
 #if !defined (private_extern)
-#  define private_extern          extern
-#endif
+#  if defined (DEBUG)
+#    define private_extern
+#  else
+#    define private_extern          extern
+#  endif
+#endif /* !defined (private_extern) */
```

Comment:
Sorry, what is this doing?

Reply:
Exposes `private_extern` methods to the tests. Unresolved symbols without this.